test/README.md

```diff
@@ -35,3 +35,90 @@ Some examples of test harnesses using these tests, from the source repository:
 - [CST parse/stringify tests](https://github.com/messageformat/messageformat/blob/11c95dab2b25db8454e49ff4daadb817e1d5b770/packages/mf2-messageformat/src/cst/cst.test.ts)
 - [Data model stringify tests](https://github.com/messageformat/messageformat/blob/11c95dab2b25db8454e49ff4daadb817e1d5b770/packages/mf2-messageformat/src/data-model/stringify.test.ts)
 - [Formatting tests](https://github.com/messageformat/messageformat/blob/11c95dab2b25db8454e49ff4daadb817e1d5b770/packages/mf2-messageformat/src/messageformat.test.ts)
+
+## Test Functions
+
+As the behaviour of `:number`, `:datetime`, and other default message functions
```

Comment:
There's a specific list. I think `:string` shouldn't be on that list?

```suggestion
As the behavior of the functions 
`:number`,
`:integer`,
`:datetime`,
`:date`,
and `:time`
in the default registry depend 
on specific versions of locale data and on their local implementation
the following functions are defined for **test use only**.
```

Reply:
Rewrote it a bit, still leaving the list open so that adding a new default registry function doesn't require updating this as well.